doc/sphinx-guides/source/installation/config.rst

```diff
@@ -248,7 +248,7 @@ this provider.
 - :ref:`:Shoulder <:Shoulder>`
 - :ref:`:IdentifierGenerationStyle <:IdentifierGenerationStyle>` (optional)
 - :ref:`:DataFilePIDFormat <:DataFilePIDFormat>` (optional)
-- :ref:`:FilePIDsEnabled <:FilePIDsEnabled>` (optional, defaults to true)
+- :ref:`:FilePIDsEnabled <:FilePIDsEnabled>` (optional, defaults to false)
```

Comment:
~Yes. We could also use flyway to change things appropriately - might still need a note somewhere but we could keep the behavior from changing.

Reply:
I think adding a flyaway script make sense, so that behavior stays the same, and a quick note to explain it.

I also think that installations that have it currently off should be set to null. But open to other ideas.